src/main/Main.ts

```diff
@@ -1,12 +1,52 @@
-import {app, ipcMain, nativeImage, BrowserWindow, screen} from "electron";
+import {app, nativeImage, BrowserWindow, screen, globalShortcut, Menu} from "electron";
 import {readFileSync} from "fs";
 import {windowBoundsFilePath} from "../utils/Common";
+import {buildMenuTemplate} from "./Menu";
+
+/**
+ * Contains references to all currently open windows.
+ *
+ * @type {Array}
+ */
+let windows: Electron.BrowserWindow[] = [];
```

Comment:
@pmkary, then maybe you'd be interested in a way to restrict the domain of the `activate` function to only accept existing elements. https://github.com/vshatskyi/black-screen/blob/3e24644dd8ec5a16f240e05291437000497588a7/src/shell/Aliases.ts#L44-L50